### ISSUE TITLE
Suggestion: meaningful PR names

### ISSUE THREAD

**Issue by hectorsector** [githubtraining/write-github-script#5]
I know we talked a little bit about PR names already, @mattdavis0351, but whether or not we validate the names of the learner's PR, I think we should definitely be suggesting meaningful PR names. This would help any learner better digest the work they're performing. If they want to accept the default _Create my-workflow.yml_ or _Update my-workflow.yml_ that's fine and their prerogative, but overall it would make a much nicer learning experience if their final repo had PRs that reflected the work they did, like _Use GitHub-script to comment on a newly created issue_ and _Use a workflow to put a new issue in triage_.

**Comment by mattdavis0351** [comment]
I agree with this, it was strictly left as default to speed up the testing of this course while also allowing for simpler validation of pull names in the config.

I still don't see why we can't just let them name it whatever we want and store than in a variable which we can reference if we need to fix mistakes

**Comment by hectorsector** [comment]
> I still don't see why we can't just let them name it whatever we want and store than in a variable which we can reference if we need to fix mistakes

The more I work with the store the more I agree with this statement. I do think we should suggest meaningful titles even if we don't validate against them.

**Comment by mattdavis0351** [comment]
@hectorsector just updated the pull titles.  Could you double check them, I'm horrible at naming this kind of thing 😛
